src/Octoshift/Services/VersionChecker.cs

```diff
@@ -21,17 +22,15 @@ public VersionChecker(HttpClient httpClient, OctoLogger log)
 
     public async Task<bool> IsLatest()
     {
-        var curVersion = GetCurrentVersion();
-        var latestVersion = await GetLatestVersion();
+        var currentVersion = Version.Parse(GetCurrentVersion());
+        var latestVersion = Version.Parse(await GetLatestVersion());
 
-        curVersion = curVersion[..latestVersion.Length];
-
-        return latestVersion.CompareTo(curVersion) <= 0;
+        return currentVersion >= latestVersion;
     }
 
     public string GetCurrentVersion()
     {
-        return Assembly.GetExecutingAssembly().GetName().Version.ToString();
+        return Assembly.GetExecutingAssembly().GetName().Version.ToString(3);
```

Comment:
This is a nice change that removes the surprising revision number in a bunch of places - including the `User-Agent` header and `generate-script` script headers. Thanks @ArinGhazarian ✨

Reply:
I could move this change into a separate PR. It's probably not really worth the effort, but up to you ❤️ 👼🏻

Reply:
I think it's good to keep it in this pr, the changes are not that much.